sphinx_rtd_theme/breadcrumbs.html

```diff
@@ -55,5 +56,14 @@
       </li>
     {% endblock %}
   </ul>
+
+  <div class="rst-breadcrumbs-buttons" role="navigation" aria-label="breadcrumb navigation">
```

Comment:
> I like the idea of options: both, top, bottom, none ;-)

Good idea. Should be a  4-enum (top bottom both None) or 2 boolean top:Yes|No bottom:Yes|No.